### PR TITLE
Fixed CC & Cohort Definition

### PR DESCRIPTION
**Pull request by aklochkova**
Fixes https://github.com/OHDSI/Atlas/issues/1338

sql scripts with constraints will be in other pr

### CODE REVIEW

File: src/main/java/org/ohdsi/webapi/cohortdefinition/dto/CohortDTO.java [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
 
 import org.ohdsi.analysis.Cohort;
+import org.ohdsi.analysis.Utils;
 import org.ohdsi.circe.cohortdefinition.CohortExpression;
 import org.ohdsi.webapi.cohortdefinition.ExpressionType;
 
 public class CohortDTO extends CohortMetadataDTO implements Cohort{
 
-    private String expression;
+    private CohortExpression expression;
```
Comment (pavgra) [inline src/main/java/org/ohdsi/webapi/cohortdefinition/dto/CohortDTO.java]: Why do you still need two mirrored fields in resulting JSON? - `expression` & `expressionStr`? Cannot we end up with a single?

Comment (aklochkova) [reply src/main/java/org/ohdsi/webapi/cohortdefinition/dto/CohortDTO.java]: I used it on UI part, now i rearranged code there and string getter became unnecessary

**Comment by pavgra** [comment]
I still observe the issue with nulls which has been [mentioned previously](https://github.com/OHDSI/WebAPI/pull/1031#issuecomment-485287651):
image

**Comment by aklochkova** [comment]
fixed on UI https://github.com/OHDSI/Atlas/pull/1700#issuecomment-486692637

**Comment by chrisknoll** [comment]
@pavgra , please flush the toilet.

**Pull request merged**